com.obsproject.Studio.Plugin.ScaleToSound.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="addon">
+  <id>com.obsproject.Studio.Plugin.ScaleToSound</id>
+  <extends>com.obsproject.Studio</extends>
+  <name>Scale To Sound Plugin</name>
+  <summary>Make a source scale reactively to the sound of an audio source</summary>
+  <url type="homepage">https://github.com/Qufyy/obs-scale-to-sound</url>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-2.0</project_license>
```

Comment:
You should add release tag so that the version of the plugin be displayed properly.